src/app/helpers/tableTemplate/pagedTableTemplate.ts

```diff
@@ -152,7 +154,7 @@ export abstract class PagedTableTemplate<T, M extends AbstractModel>
       );
   }
 
-  protected apiAction(filters: Filters<M>, args: AbstractModel[]) {
+  protected apiAction(filters: Filters<M>, args?: AbstractModel[]) {
```

Comment:
I suspect this should instead be `protected apiAction(filters: Filters<M>, args: AbstractModel[] = []) {`